test/check-machines-storage-pools

```diff
@@ -748,6 +748,238 @@ class TestMachinesStoragePools(VirtualMachinesCase):
+                self.activate = activate
+                self.remove_volume = remove_volume
+                self.xfail = xfail
+                self.xfail_error = xfail_error
+
+            def execute(self):
+                self.create()
+                self.delete()
+                self.verify()
+
+            def create(self):
+                create_cmd = f"virsh pool-define-as {self.name} --type {self.pool_type} --target {self.target} "
+                if self.pool_type == "netfs":
+                    create_cmd += f"--source-host {self.source_host} --source-path {self.source_path}"
+                elif self.pool_type == "iscsi":
+                    create_cmd += f"--source-host {self.source_host} --source-dev {self.source_dev}"
+                elif self.pool_type == "disk":
+                    create_cmd += f"--source-dev {self.source_dev} --source-format {self.source_format}"
+                else:
+                    raise AssertionError("unsupport pool_type")
+
+                m.execute(create_cmd)
+
+                # Check whether the pool creation is successful
+                wait(lambda: self.name in m.execute("virsh pool-list --all"))
+
+                if self.activate:
+                    m.execute(f"virsh pool-start {self.name}")
+
+                    b.wait_text(f"#pool-{self.name}-{self.connection}-state", "active")
+
+                    if self.pool_type == "iscsi":
+                        wait(lambda: "unit:0:0:0" in m.execute(f"virsh pool-refresh {self.name} && virsh vol-list {self.name}"))
+                    elif self.vol_name:
+                        if self.vol_name not in m.execute(f"virsh vol-list {self.name}"):
+                            m.execute(f"virsh vol-create-as {self.name} {self.vol_name} 1M")
+                            wait(lambda: self.vol_name in m.execute(f"virsh vol-list {self.name}"))
+                            # need refresh after volume creation
+                            b.reload()
+                            b.enter_page('/machines')
+
+                self.test_obj.waitPoolRow(self.name)
+                self.test_obj.togglePoolRow(self.name)
+
+            def delete(self):
+                # Expand and click the delete button
+                b.click(f"#pool-{self.name}-{self.connection}-action-kebab button")
+                b.click(f"#delete-pool-{self.name}-{self.connection}")
+
+                # Check if deletion dialog is shown
+                b.wait_visible("div.pf-c-modal-box")
+
+                if self.activate:
+                    if self.vol_name or self.pool_type == "iscsi":
+                        b.set_checked("#storage-pool-delete-volumes", self.remove_volume)
+                else:
+                    # Check the content of inactive pool deletion dialog
+                    b.wait_text(".pf-c-form.pf-m-horizontal", "Deleting an inactive storage pool will only undefine the pool. Its content will not be deleted.")
```

Comment:
> Sure, I will check them later and re-push, thanks. And I also check the failed jobs, it seems that
"[0523/065157.086428:ERROR:cert_verify_proc_builtin.cc(681)] CertVerifyProcBuiltin for 127.0.0.2 failed:
----- Certificate i=0 (CN=fedora-36-127-0-0-2-2201,O=bcc8400d49b44879b9e69d2a326a8378,C=US) -----
ERROR: No matching issuer found" is the reason why the case failed, " shows the reason why the cases failed. Do you have any ideas about that?

Where do you see that error? I only see tests failing with the error:
`wait_js_cond(ph_is_present(".pf-c-form.pf-m-horizontal")): Uncaught (in promise) Error: condition did not become true`
Which should be fixed by this change:
```suggestion
                    b.wait_text(".pf-c-form .pf-m-horizontal", "Deleting an inactive storage pool will only undefine the pool. Its content will not be deleted.")
```

Reply:
The error could be found in results of debian-stable, fedora 35, fedora 36 and etc. It seems that results of rhel-8.7 and rhel-9.1 doesn't contain this error. 

Also, thanks to point the selector problem, I have modify it locally, and will push it later